db/migrate/20211104065354_reset_datastream_revisions.rb

```diff
@@ -0,0 +1,9 @@
+class ResetDatastreamRevisions < ActiveRecord::Migration[5.2]
+  def up
+    Revision.find_by(name: 'datastreams')&.delete
```

Comment:
Is this enough for a proper force import?

Reply:
We also need the ENV var, but last time it worked :wink: